hw3/01-characters.js

```diff
@@ -0,0 +1,39 @@
+// url for the Thrones API
+const url = "https://thronesapi.com/api/v2/Characters";
+const loader = document.getElementById("loading");
+
+const fetchThrones = async (thronesUrl) => {
+  try {
+    const response = await fetch(Thronesurl);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    console.error("Error:", error);
+    return [];
+  }
+};
+
+const runFunThrones = async () => {
+  const nonAlcoholicDrinks = await fetchThrones(url);
```

Comment:
Does the "fetchThrones()" function actually return a promise?

Why is "await" used here?